ios/src/platform_ios.mm

```diff
@@ -1,15 +1,21 @@
 #ifdef PLATFORM_IOS
 
 #import <Foundation/Foundation.h>
```

Comment:
Since you're importing UIKit below, you can remove the foundation import (UIKit includes it).